src/Explorer/Controls/NotebookGallery/InfoComponent/InfoComponent.tsx

```diff
@@ -25,6 +29,11 @@ export class InfoComponent extends React.Component {
         <Stack.Item>
           {this.getInfoPanel("KnowledgeArticle", "Microsoft Terms of Use", CodeOfConductEndpoints.termsOfUse)}
         </Stack.Item>
+        {this.props.onReportAbuseClick !== undefined && (
+          <Stack.Item>
+            {this.getInfoPanel("ReportHacked", "Report Abuse", undefined, () => this.props.onReportAbuseClick())}
```

Comment:
>ReportHacked [](start = 32, length = 12)

?

Reply:
"ReportHacked" is actually the icon name in Office UI Fabric Icons pack that we're using here.
https://uifabricicons.azurewebsites.net/